mobile/cmp/navigator/NavigatorModel.js

```diff
@@ -258,4 +276,48 @@ export class NavigatorModel extends HoistModel {
         this._callback = null;
     }
 
+    @action
```

Comment:
All handlers can be defined as fat-arrow functions properties on the model -- avoids need for extra wrapper in render method.  This something we have been doing more and more of.

Reply:
should we be consuming the event when we accept it (e.g. e.stopPropogation(); e.preventDefault()).  Would that potentially circumvent the browser grabbing the event?  In any case, probably a best practice.